### DISCUSSION TITLE
Encouragement to do better

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#343] [Ideas]
Every time you miss the keyboard injects your fingertips with fentanyl

yes this is a joke

**Comment by NexIsDumb** [comment]
amazing

**Comment by r6915ee** [comment]
> Every time you miss the keyboard injects your fingertips with fentanyl
> 
> yes this is a joke

peak idea actually not gonna lie
